site/docs/components/stepper-input/examples.mdx

```diff
@@ -0,0 +1,133 @@
+
+### Best practices
+
+The number of decimal places should be suitable for the specific use case. For instance, avoid accepting three decimal places when a retail user is dealing with currencies like pounds sterling or U.S. dollars, which typically require only two decimal places.
+
+</LivePreview>
+<LivePreview componentName="stepper-input" exampleName="Limits" >
+
+## Min and max value
+
+If you need to limit the range of accepted values, you’re able to set a maximum or minimum (or both).
+
+### Best practices
+
+- Clearly state the accepted value range in the form field’s helper text.
+- Use the form field error state to inform users who manually type in a value outside the accepted range that the value is invalid.
+
+</LivePreview>
+<LivePreview componentName="stepper-input" exampleName="StepAndBlock" >
+
+## Custom steps
+
+Customize the default, increment and block values to suit your use case.
+
+### Best practices
+
+- Pick a default value that reflects the desired value for most users, such as the most frequently submitted number.
+- Make sure you set an increment value that’s appropriate for the context. For example, if the user needs to enter ”Number of people”, adjustments of 0.5 would be confusing. If you’re undecided, consider using an increment value of 1.
+
+</LivePreview>
+<LivePreview componentName="stepper-input" exampleName="ButtonAdornments" >
+
+## Button adornments
+
+You can use [Button](/salt/components/button) at the end of the input field with the `endAdornment` prop. Buttons can allow for custom interactivity within the input field itself. Examples include:
+
+**Refresh adornment**:
+
+Reset the input to it’s default value, using the refresh button.
+
+**Sync adornment**:
+
+Sync to a live value from a source that is constantly changing, for example, when dealing with live stock prices. The component's value doesn't automatically update when the live value changes, however. In this case, the sync button automatically appears when the displayed value and live value don’t match.
+
+**Custom buttons**:
+
+Customize the positioning and appearance of the stepper buttons to meet specific product requirements or restrictions.
+
+</LivePreview>
+<LivePreview componentName="stepper-input" exampleName="Disabled" >
+
+## Disabled
+
+You can disable a stepper input, resulting in no action when the user interacts with it.
+
+</LivePreview>
+<LivePreview componentName="stepper-input" exampleName="Readonly" >
+
+## Read-only
+
+You can set stepper input to a read-only state. The user can't edit the value text in this state, but they can highlight and copy it.
+
+By default, the marker is an em dash. If your application requires a different marker, you can set it via the `emptyReadOnlyMarker` prop.
+
+- Use the read-only state when the user needs the value for their flow or current task but cannot edit it, for example, because of user permissions.
```

Comment:
Is this a best practice? There are some short lists without headings (this example and the next one)